### PR TITLE
Add the readme updates for all new configurations

### PR DESCRIPTION
**Pull request by found-it**
Signed-off-by: James Petersen <jpetersenames@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,52 +213,215 @@ log:
...
-mode: periodic
+mode: adhoc
 
 # Only respected if mode is periodic
 polling-interval-seconds: 300
+```
+
+### Missing Tag Policy
+
+There are cases where image in kubernetes do not have an associated tag - for
```
Comment (subecho) [inline README.md:313]:
```suggestion
There are cases where images in Kubernetes do not have an associated tag - for
```

File: README.md [hunk 2]
```diff
@@ -107,52 +213,215 @@ log:
...
+example when an image is deployed using the digest
```
Comment (subecho) [inline README.md:314]:
```suggestion
example when an image is deployed using the digest.
```

File: README.md [hunk 3]
```diff
@@ -14,66 +14,167 @@ It can be run inside a cluster (under a Service Account) or outside (via any pro
 ## Installation
 Kai can be run as a CLI, Docker Container, or Helm Chart
 
-By default, Kai will look for a Kubeconfig in the home directory to use to authenticate (when run as a CLI). 
+By default, Kai will look for a Kubeconfig in the home directory to use to authenticate (when run as a CLI).
```
Comment (subecho) [inline README.md:17]: Guess we should standardize on KAI for the name of the project 👍

**Review by subecho — COMMENTED** [review]
Just a couple of quick suggestions but I think we'll be good to go once those are in! Thanks, James! 👍

File: README.md (hunk 3 above)
Comment (found-it) [reply README.md]: Done!

**Review by zhill — APPROVED** [review]
lgtm. I'd generally prefer to move releasing into its own file, but not a big deal at all and we can update that later if needed.

**Pull request merged**